### PR TITLE
Added a paragraph about using only Annotations as mapping type

### PR DESCRIPTION
**Pull request by IceShack**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  |no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documented?   | yes
| License       | MIT

I had some issues figuring out how the Annotations work without using any other type of mapping. It says nowhere that you would need an empty RootQuery/RootMutation.
This should clear things up.

### CODE REVIEW

File: docs/annotations/index.md [hunk 1]
```diff
@@ -14,6 +14,34 @@ overblog_graphql:
...
+namespace App\GraphQL\Query;
+
+/**
+ * @GQL\Type
+ */
+class RootQuery
+{
+}
+```
+If you use mutations, you need a RootMutation aswell.
```
Comment (ooflorent) [inline docs/annotations/index.md]:
```suggestion
If you use mutations, you need a `RootMutation` type as well.
```

File: docs/annotations/index.md [hunk 2]
```diff
@@ -14,6 +14,34 @@ overblog_graphql:
...
+  definitions:
+    schema:
+      query: RootQuery
+    mappings:
+      types:
+        - type: annotation
+          dir: "%kernel.project_dir%/src/GraphQL"
+          suffix: ~
+```
+Your RootQuery class should look like this:
```
Comment (ooflorent) [inline docs/annotations/index.md]:
```suggestion
Your `RootQuery` class should look like this:
```

File: docs/annotations/index.md [hunk 3]
```diff
@@ -14,6 +14,34 @@ overblog_graphql:
 
 This will load all annotated classes in `%kernel.project_dir%/src/GraphQL` into the schema.
 
+## Use Annotations as your only Mapping
+
+If you only use annotations as mappings you need to add an empty RootQuery type.
```
Comment (ooflorent) [inline docs/annotations/index.md]:
```suggestion
If you only use annotations as mappings you need to add an empty `RootQuery` type.
```

File: docs/annotations/index.md [hunk 4]
```diff
@@ -14,6 +14,34 @@ overblog_graphql:
 
 This will load all annotated classes in `%kernel.project_dir%/src/GraphQL` into the schema.
 
+## Use Annotations as your only Mapping
```
Comment (murtukov) [inline docs/annotations/index.md]:
```suggestion
## Using Annotations as your only Mapping
```

**Comment by mcg-web** [comment]
Thank you @IceShack !

**Pull request merged**